Resources/public/js/models/ez-locationmodel.js

```diff
@@ -180,6 +180,27 @@ YUI.add('ez-locationmodel', function (Y) {
         },
 
         /**
+         * Swap the location with other location
+         * 
+         * @method swap
+         * @param {Object} options the required for the swap
+         * @param {eZ.Location} Location where the given one will be relocated
```

Comment:
* Order should be fixed: `options.api` should be right after `options`
* Missing name of varible: `@param {eZ.Location} destinationLocation`